src/containers/Explore.tsx

```diff
@@ -3,50 +3,159 @@ import * as React from 'react';
+    if (api != null) {
+      docsDom = this.renderApiDocs(api);
+      deprecated = this.renderDeprecationWarning(api);
     }
 
     if (docsDom == null) {
-      docsDom = this.renderIndex();
+      docsDom = <ExplorePage />;
     }
+
+    if (category) {
+      header = <PageHeader id="api-documentation" halo={category.name} header={api!.name} />
+    }
+
     return (
       <div role="region" aria-labelledby="api-documentation">
-        <h1 id="api-documentation">API Documentation</h1>
+        {header}
+        {deprecated}
         {docsDom}
       </div>
     );
   }
 
-  private renderIndex() {
-    return <ExplorePage />;
+  private renderDeprecationWarning(apiDefinition: IApiDescription) {
+    const { deprecationContent } = apiDefinition;
+
+    if (!deprecationContent) {
+      return null;
+    }
+
+    return (
+      <div className="usa-alert usa-alert-info">
+        <div className="usa-alert-body">
+          {deprecationContent({})}
+        </div>
+      </div>
+    );
+  }
+
+  private renderApiDocs(apiDefinition: IApiDescription) {
+    let docs: JSX.Element | null = null;
+    // because this is downstream from a getApi() call, we can assert that apiName is defined
+    const apiName : string = this.props.match.params.apiName!;
+    const category = lookupApiCategory(this.props.match.params.apiCategoryKey);
+    if (apiDefinition.docSources.length === 1) {
+      docs = <SwaggerDocs docSource={apiDefinition.docSources[0]} apiName={apiName} />;
+    } else {
+      docs = (
+        <React.Fragment>
+          {category!.tabBlurb}
+          <Tabs selectedIndex={this.state.tabIndex} onSelect={tabIndex => this.setState({ tabIndex })}>
+            <TabList>
+              {apiDefinition.docSources.map(apiDocSource => {
+                return (
+                  <Tab key={apiDocSource.label}>
+                    {apiDocSource.label}
+                  </Tab>
+                );
+              })}
+            </TabList>
+            {apiDefinition.docSources.map(apiDocSource => {
+              return (
+                <TabPanel key={apiDocSource.label}>
+                  <SwaggerDocs docSource={apiDocSource} apiName={apiName} />
+                </TabPanel>
+              );
+            })}
+          </Tabs>
+        </React.Fragment>
+      )
+    }
+
+    return (
+      <Flag name={`hosted_apis.${apiDefinition.urlFragment}`}>
+        {docs}
+      </Flag>
+    )
+  }
+
+  private getApi() : IApiDescription | null {
+    if (!this.props.match.params.apiName) {
+      return null;
+    }
+
+    return lookupApiByFragment(this.props.match.params.apiName);
+  }
+
+  private setTabIndexFromHash() : void {
+    const api = this.getApi();
+    if (api !== null && api.docSources.length > 1) {
+      const newTabIndex = this.getTabIndexFromHash(api.docSources);
+      this.setState({ tabIndex: newTabIndex });
+    }
+  }
+
+  private getTabIndexFromHash(apiDocSources : IApiDocSource[]) : number {
```

Comment:
```suggestion
  private getTabIndexFromFragment(apiDocSources : IApiDocSource[]) : number {
```

and associated changes